app/instance-initializers/ember-href-to.js

```diff
@@ -0,0 +1,39 @@
+import Em from 'ember';
+
+function _getNormalisedRootUrl(router) {
+  var rootURL = router.rootURL;
+  if(rootURL.charAt(rootURL.length - 1) !== '/') {
+    rootURL = rootURL + '/';
+  }
+  return rootURL;
+}
+
+let router;
+
+export default {
+  name: 'ember-href-to',
+  initialize: function(applicationInstance) {
+    router = applicationInstance.container.lookup('router:main');
```

Comment:
It prevents `Assertion Failed: calling set on destroyed object` errors, I'm not immediately sure why